wireapp/kalium
# fix(user-delete): verify handle delete user event

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?
Added verification to handling delete user event.
### Issues

We received the delete user event even for our connections, we didn't verify the user id before logging the current user off.

### Causes (Optional)
We didn't verify the user id with the current session user id.

### Solutions
Compare the currentSession userId with the userId received within the event.

Needs releases with:


### Testing
* Create two accounts
* Connect them and start a conversation
* delete one of the accounts
* only the deleted account clients should be logged out

#### Test Coverage (Optional)


#### How to Test

_Briefly describe how this change was tested and if applicable the exact steps taken to verify that it works as expected._

### Notes (Optional)

_Specify here any other facts that you think are important for this issue._

### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
